gp_emulator/gpu/kernel_rowSum.cu

```diff
@@ -0,0 +1,29 @@
+/*********************************************//**
+ * row sum:
+ * return vector rowSum(A)
+ *********************************************/
+#include "gpu_predict.h"
+
+real* gpu_rowSum(const real *A, const int A_nrows,const int A_ncols)
+{
+    cublasStatus_t stat;
+    cublasHandle_t handle;
+    stat=cublasCreate(&handle);
+
+    real alpha = 1.f;
```

Comment:
In this case, this alpha is a parameter of cublas gemv. The value is a scalar which multiplies the vector.